standalone/test/src/communication_layer/scanner_mock.cpp

```diff
@@ -121,4 +125,14 @@ void ScannerMock::sendEmptyMonitoringFrame()
   data_server_.asyncSend(monitoring_frame_receiver_, data);
 }
 
+void ScannerMock::sendMonitoringFrames(const std::vector<data_conversion_layer::monitoring_frame::Message>& msgs)
+{
+  for (const auto& msg : msgs)
+  {
+    sendMonitoringFrame(msg);
+    // Sleep to ensure that message are not sent too fast which might cause messages overwrite in socket buffer
```

Comment:
Could you elaborate a bite more?

Reply:
This was originally added by Hagen in Add DroppedMsgDetector (#81). I don't know the details.